src/extensions/sessionrecording.ts

```diff
@@ -22,6 +22,7 @@ import { logger, loadScript, _timestamp, window } from '../utils'
 const BASE_ENDPOINT = '/s/'
 
 export const RECORDING_IDLE_ACTIVITY_TIMEOUT_MS = 5 * 60 * 1000 // 5 minutes
+export const THIRTY_MINUTES_IN_MS = 30 * 60 * 1000
```

Comment:
This makes total sense... I even wonder if we shouldn't make it slightly lower, like say 10 mins?